test/cppsuite/configs/cursor_microbenchmarks_default.txt

```diff
@@ -0,0 +1,25 @@
+# Configuration for cursor_microbenchmarks.
+duration_seconds=1,
+cache_size_mb=2000,
+validate=false,
+in_memory=true,
+sweep_interval=1000,
+workload_manager=
+(
+    populate_config=(
+        collection_count=1,
+        key_count_per_collection=1,
+        key_size=4,
+        value_size=1
+    ),
+    custom_config=
```

Comment:
General cppsuite question, what's the difference between `populate_config` and `custom_config`? I can gather there's an initial populate phase but I don't know where the `custom_config` applies.

Reply:
`custom_config` is dedicated to a custom operation that is defined in the workload, look for `custom_operation`.

Reply:
Got it - thanks.